client.go

```diff
@@ -254,12 +254,12 @@ func (cli *S3) CheckPartUploaded(ctx context.Context, req *UploadPartRequest) (b
 
 	for _, part := range parts {
 		if *part.PartNumber == int64(req.ChunkNumber) {
-			log.Event(ctx, "chunk number already uploaded", log.INFO, log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName, "identifier": req.UploadKey})
+			log.Info(ctx, "chunk number already uploaded", log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName, "identifier": req.UploadKey})
 			return true, nil
 		}
 	}
 
-	log.Event(ctx, "chunk number failed to upload", log.ERROR, log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName})
+	log.Error(ctx, "chunk number failed to upload", nil, log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName})
```

Comment:
I'm wondering if we're meant to always pass an error when using `log.Error` (instead of `nil`)?

Reply:
Fixed to pass sensible error in those two cases. Thanks!